src/modules/mc_pos_control/MulticopterPositionControl.cpp

```diff
@@ -477,6 +477,12 @@ void MulticopterPositionControl::start_flight_task()
 		return;
 	}
 
+	// Switch to clean new task when mode switches e.g. to reset state when switching between auto modes
+	if (_last_vehicle_nav_state != _vehicle_status.nav_state
+	    && _vehicle_status.nav_state != vehicle_status_s::NAVIGATION_STATE_ORBIT) {
```

Comment:
Add a quick mention why ORBIT is excluded?

Reply:
Yes ✔️